### ISSUE TITLE
The WakeWordAgent is not starting up.

### ISSUE THREAD

**Issue by Partha2k** [alexa/alexa-avs-sample-app#685]
Still now I haven't experimented with any of the mic configurations yet, that's the reason I am getting the wakewordAgent error.Can A USB Sound Card along with 3.5mm Jacked Mic be used to work the wakeWordAgent kitt.ai rather than a USB Mic?

**Comment by dskidmor** [comment]
Yes, a USB sound card with mic input should work fine, provided your computer is able to recognize it. What kind of error are you getting?

**Comment by Partha2k** [comment]
![screenshot 45](https://cloud.githubusercontent.com/assets/25670426/25066012/ee872c52-2237-11e7-98d5-9d41a75fd049.png)

@dskidmor I am getting this error when starting up the kitt_ai.

**Comment by Steakschen** [comment]
got this error too

Failed to open portaudio stream

**Comment by sginsbe** [comment]
I have the same error, sensory WakeWordAgent works fine but kitt.ai will not start.

**Comment by scotthea-amazon** [comment]
Hello,
It looks like you may not have not configured the recording device in ALSA correctly.  Would you please post the ALSA configuration you are using (it should be in ~/.asoundrc), and the output from running the command: "arecord -l".
-SWH

**Comment by Partha2k** [comment]
@scotthea-amazon yeah I actually selecting the boards default sound card. After choosing the usb sound card it worked fine.

**Comment by Steakschen** [comment]
How can i change to the other soundcard?

**Comment by sginsbe** [comment]
This is what I get as the output of arecord-l

`**** List of CAPTURE Hardware Devices ****
card 1: Device [USB PnP Sound Device], device 0: USB Audio [USB Audio]
  Subdevices: 0/1
  Subdevice #0: subdevice #0
`
It seems like the USB soundcard is selected, unless I'm misunderstanding this?

**Comment by Partha2k** [comment]
@Steakschen open alsamixer(alsamixer) select the sound card which is inserted into USB then on another terminal window type in alsactl(sudo alsactl store 1) to save the current changes. To test the mic open the arecord test file (arecord -D plughw:1 --duration=10 -f cd -vv ~/rectest.wav) it will record 10 secs of audio. For playback of the recorded audio type in (aplay ~/rectest.wav) [~/: this may change according to your rectest file location]

**Comment by Steakschen** [comment]
@Partha2k Thank you i will try
